### PR TITLE
Use sidechain pallet and update to polkadot-v0.9.20

### PR DESCRIPTION
**Pull request by gaudenzkessler**
* imports the pallet_sidechain
* updates substrate to polkadot-v0.9.20 to align with the pallet repo

https://github.com/integritee-network/pallets/pull/51

### CODE REVIEW

**Comment by gaudenzkessler** [comment]
the pallet branch is for the draft. After pallets repo is merged, this will be reset to "master".

File: node/src/command.rs [hunk 1]
```diff
@@ -146,17 +144,24 @@ pub fn run() -> sc_cli::Result<()> {
 
 						cmd.run::<Block, service::ExecutorDispatch>(config)
 					},
-					BenchmarkCmd::Block(cmd) => cmd.run(client),
+					BenchmarkCmd::Block(cmd) => {
+						let PartialComponents { client, .. } = service::new_partial(&config)?;
```
Comment (haerdib) [inline node/src/command.rs:148]: What's the reason for this change?

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -15,52 +15,53 @@ targets = ["x86_64-unknown-linux-gnu"]
...
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
+frame-system-benchmarking = { version = "4.0.0-dev", default-features = false,  optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
 hex-literal = { version = "0.3.1", optional = true }
 
 [build-dependencies]
-substrate-wasm-builder = {version = "5.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19" }
+substrate-wasm-builder = {version = "5.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
 
 [features]
 default = ["std"]
```
Comment (haerdib) [inline runtime/Cargo.toml:67]: Can you add the pallet-sidechain to the std feature?

File: runtime/src/weights/pallet_sidechain.rs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+//! Autogenerated weights for `pallet_teerex`
```
Comment (haerdib) [inline runtime/src/weights/pallet_sidechain.rs:1]: @clangenb Should we keep the weights of a test-node up to date, or can be we leave it like this?

**Review by haerdib — CHANGES_REQUESTED** [review]

**Comment by haerdib** [comment]
Can you update the PR description to `update to polkadot-v0.9.20` and add it to the PR title? :)

File: runtime/src/weights/pallet_sidechain.rs (hunk 3 above)
Comment (clangenb) [reply runtime/src/weights/pallet_sidechain.rs:1]: We can leave it like this, as long as we did not change the logic of the functions drastically, the weights will stay similar anyhow.

File: node/src/command.rs (hunk 1 above)
Comment (gaudenzkessler) [reply node/src/command.rs:148]: Substrate changed the Benchmarks: https://github.com/paritytech/substrate/blob/563f4820d8f36d256ada7ea3fef46b2e94c4cd5a/bin/node-template/node/src/command.rs#L126

File: runtime/Cargo.toml (hunk 2 above)
Comment (gaudenzkessler) [reply runtime/Cargo.toml:67]: One day I will learn to do it without reminder :)

File: node/src/command.rs (hunk 1 above)
Comment (haerdib) [reply node/src/command.rs:148]: I see. Thanks for the info

**Review by haerdib — APPROVED** [review]
LGTM!

**Review by clangenb — COMMENTED** [review]
LGTM!

**Review by clangenb — APPROVED** [review]
